lib/Sema/TypeCheckAttr.cpp

```diff
@@ -2862,7 +2862,7 @@ void AttributeChecker::visitCustomAttr(CustomAttr *attr) {
 
   // If the nominal type is a property wrapper type, we can be delegating
   // through a property.
-  if (nominal->getPropertyWrapperTypeInfo()) {
+  if (nominal->getAttrs().hasAttribute<PropertyWrapperAttr>()) {
```

Comment:
What does this change do?

Reply:
This change only suppresses the `cannot be used as attribute` diagnostic in the case where the nominal type for the custom attribute type is declared as `@propertyWrapper` but there's some other error, such as the type not having a non-static `wrappedValue` property